### ISSUE TITLE
Leetcode submit fails

### ISSUE THREAD

**Issue by dmartyanov** [skygragon/leetcode-cli#177]
leetcode submit fails for me 
```
~/p/c/l/leetcode-samples> leetcode submit 174.dungeon-game.py
/Users/dmartyanov/psn/contests/leetcode/leetcode-cli/lib/plugins/leetcode.js:264
    x.stdout = util.inspect((result.code_output || []).join('\n'));
                                                       ^

TypeError: (result.code_output || []).join is not a function
    at formatResult (/Users/dmartyanov/psn/contests/leetcode/leetcode-cli/lib/plugins/leetcode.js:264:56)
    at Request._callback (/Users/dmartyanov/psn/contests/leetcode/leetcode-cli/lib/plugins/leetcode.js:233:16)
    at Request.self.callback (/Users/dmartyanov/psn/contests/leetcode/leetcode-cli/node_modules/request/request.js:185:22)
    at Request.emit (events.js:209:13)
    at Request.<anonymous> (/Users/dmartyanov/psn/contests/leetcode/leetcode-cli/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:209:13)
    at IncomingMessage.<anonymous> (/Users/dmartyanov/psn/contests/leetcode/leetcode-cli/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:297:20)
    at IncomingMessage.emit (events.js:214:15)
    at endReadableNT (_stream_readable.js:1178:12)
```

CLI v2.6.1-8dc0b2a
[Environment]
Node                 v12.9.0
OS                   darwin 18.7.0
[Plugins]
github               2018.11.18

**Comment by ember526** [comment]
Same thing here, maybe we are running into some serious problem

**Comment by 9527KGG** [comment]
Any suggestion when the fix could be released? Currently, it's just so inconvenient. Thanks.

**Comment by skygragon** [comment]
@9527KGG the fresh released 2.6.2 should fix this, sorry for this late reply
